examples/Cassie/cassie_rbt_state_estimator.cc

```diff
@@ -1204,6 +1206,20 @@ EventStatus CassieRbtStateEstimator::Update(const Context<double>& context,
                             &right_contact);
     }
 
+    // Test mode needed for hardware experiment
```

Comment:
done

Reply:
I think this could still be slightly clearer - maybe a function in eigen_utils that's called RectifyQuaternion() or NormalizeQuaternion() that does this check and fix?

Reply:
I don't understand how this line is related to adding(?) a function for quaternion

Reply:
Oh i must have been referencing old code